Wikipedia/Code/SavedArticlesFetcher.m

```diff
@@ -140,9 +140,17 @@ - (void)fetchTitle:(MWKTitle*)title {
         self.fetchOperationsByArticleTitle[title] =
             [self.articleFetcher fetchArticleForPageTitle:title progress:NULL].thenOn(self.accessQueue, ^(MWKArticle* article){
             @strongify(self);
-            return [self downloadImageDataForArticle:article].thenOn(self.accessQueue, ^{
-                [self didFetchArticle:article title:title error:nil];
```

Comment:
@joewalsh ya that looked good, i was just unsure how formerly errors were handled. my promise-foo is weak! :)